master/master.cfg

```diff
@@ -191,6 +191,7 @@ nogate_builders = [
     "auto-bitrig-64-opt", 
     "auto-linux-musl-64-opt",
     "auto-win-msvc-64-opt",
+    "cargo-bitrig-64"
```

Comment:
I think that this array is only used for the auto builders themselves, not necessarily the cargo builders unfortunately. I also unfortunately am not familiar with how `nogate_builders` works :(